DESCRIPTION

```diff
@@ -3,17 +3,21 @@ Title: Lake Physics Tools
 Maintainer: Luke Winslow <lawinslow@gmail.com>
 Version: 1.9.1

```

Comment:
Could you increment the minor version. We didn't make any codebreaking changes, but you added some significant functionality. I'd say 1.10.0

Reply:
Addressed with https://github.com/GLEON/rLakeAnalyzer/pull/85/commits/cc923ebde1bcf10e86b7645b78390dc6fc94fe22